src/services/services.ts

```diff
@@ -6772,6 +6762,67 @@ namespace ts {
             return [];
         }
 
+        /**
+         * Checks if position points to a valid position to add JSDoc comments, and if so,
+         * returns the appropriate scaffolding. Otherwise returns an empty string.

```

Comment:
Do any of these alternatives work?
* frame
* skeleton
* outline (already used for another feature, we should avoid a conflict)
* blueprint
* sketch
* basis/base
* shell
* mold
* foundation

Of these, I think "skeleton" is best. What do you think @DanielRosenwasser?